packages/tldraw/src/test/frames.test.ts

```diff
@@ -234,6 +256,58 @@ describe('frame shapes', () => {
 			h: 50,
 		})
 	})
+	it('Unparents a shape when resize causes it to be out of bounds', () => {
+		const rectId: TLShapeId = createRect({ pos: [70, 10] })
+		dragCreateFrame()
+		// resize the frame so the shape is out of bounds
+		editor.pointerDown(100, 50, { target: 'selection', handle: 'right' })
+		editor.pointerMove(50, 50)
+		editor.pointerUp(50, 50)
+		const parent = editor.getShape(rectId)?.parentId
+		expect(parent).toBe('page:page')
+	})
+
+	it('doesnt unparent a shape that is only partially out of bounds', () => {
+		const rectId: TLShapeId = createRect({ pos: [70, 10] })
+		const frameId = dragCreateFrame()
```

Comment:
I'd probably add an expect below this line which makes sure that the rect is parented to the frame. Makes it a bit more explicit what change we expect to occur.